main.py

```diff
@@ -1,10 +1,36 @@
 import init_django_orm  # noqa: F401
-
+from json import load
```

Comment:
Prefer using `import json` and calling load as `json.load` to be more descriptive in the code

Reply:
Ok, but I thought that it is better not to import the whole module, and just import the method that is needed is better.